src/components/people/PersonCard/index.tsx

```diff
@@ -30,11 +38,12 @@ export default ({
     photoSize = 'xlarge',
     hidePopover,
     showJobTitle,
+    isFetchingPerson,
 }: PersonCardProps) => {
     const [currentPerson, setCurrentPerson] = useState<PersonDetails>();
-    const { error, personDetails } = personId
+    const { isFetching, error, personDetails } = personId
```

Comment:
This condition will cause a inconsistent hook call, usePersonDetails will only be called if personId is set. This is against the [rules of hooks](https://reactjs.org/docs/hooks-rules.html )

Reply:
This behavior is already in master and can be fixed in a separate pr